sdk/tests/conformance/extensions/webgl-draw-buffers.html

```diff
@@ -553,17 +560,30 @@
   if (maxUsable > 1) {
     var bufs1 = makeColorAttachmentArray(maxUsable);
     var bufs2 = makeColorAttachmentArray(maxUsable);
+    var bufs3 = makeColorAttachmentArray(maxUsable);
+    var bufs4 = makeColorAttachmentArray(maxUsable);
```

Comment:
How about a one-line comment before each "bufs" variable saying what each intends to test? (For example, "First half of color buffers disabled", "Second half", etc.)